src/custom-types.json

```diff
@@ -236,6 +236,14 @@
   "application/x-7z-compressed": {
     "compressible": false
   },
+  "application/x-arj": {
+    "compressible": false,
+    "extensions": ["arj"],
+    "sources": [
+      "https://en.wikipedia.org/wiki/List_of_archive_formats",
+      "http://www.wussu.com/various/mimetype.htm"
```

Comment:
When you get a chance, can you add a link to where this is defined, please note that

> This needs to be a [primary source](https://en.wikipedia.org/wiki/Primary_source);
  links to type aggregating sites and Wikipedia are _not acceptible_.

Reply:
Sorry, I didn't merge this PR since it had two types with issues. The ARJ one was resolved, but the ANS one seems less likely to be. To that end, I am going to merge just the ARJ part of the PR and the ANS change can be re-opened as a new PR if it is desired to work through.

Reply:
@dougwilson Sounds great, thank you!